app/src/main/res/navigation/navigation_main.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I'd recommend creating the navigation graph programmatically, it looks like Google is trying to move away from XML